ts/components/session/ActionsPanel.tsx

```diff
@@ -43,29 +52,46 @@ export enum SectionType {
   Channel,
   Settings,
   Moon,
+  PathIndicator,
 }
 
-const Section = (props: { type: SectionType; avatarPath?: string }) => {
+const Section = (props: {
+  setModal?: any;
+  type: SectionType;
+  avatarPath?: string;
+  hasOnionPath?: boolean;
+}) => {
   const ourNumber = useSelector(getOurNumber);
   const unreadMessageCount = useSelector(getUnreadMessageCount);
   const theme = useSelector(getTheme);
   const dispatch = useDispatch();
-  const { type, avatarPath } = props;
+  const { setModal, type, avatarPath, hasOnionPath } = props;
```

Comment:
if `hasOnionPath` is already in redux you should be able to just use it with a `useSelector`